src/CMakeLists.txt

```diff
@@ -7,6 +7,13 @@ project(RevBayes)
 
 set(CMAKE_CXX_STANDARD 17)
 
+# Congifure compiler to support big objects
+if(MSVC)
+  add_compile_options(/W4 /EHsc /bigobj)
+else()
```

Comment:
I think you need to gate this on WIN32.  Otherwise it breaks non-windows builds.

So something like `elseif(WIN32)` perhaps.

Reply:
I gated the whole block – I figured it was more conservative, even if few non-windows users are likely to be using MSVC